default/AI/ShipDesignAI.py

```diff
@@ -1445,7 +1404,7 @@ def _rating_function(self):
         if self.troops == 0:
             return INVALID_DESIGN_RATING
         else:
-            return self.troops/(self.production_cost**((1+foAI.foAIstate.shipCount * AIDependencies.SHIP_UPKEEP)**-1))
+            return self.troops/self._adjusted_production_cost()
```

Comment:
For me `return a/b()` looks same as in example `i=i+1`.  This situation is not described in PEP8, but adding whitespaces around single operator is  common practice in python documentation (See example in my first comment).